zitadel/helper/client.go

```diff
@@ -9,6 +9,7 @@ import (
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/zitadel/oidc/v3/pkg/oidc"
+	"github.com/zitadel/zitadel-go/v3/pkg/client/action/v2beta"
```

Comment:
should we use the v2 API? We might have to bump the API version again though.

Reply:
Really valid point. I'll make the change

Reply:
Hi @eliobischof I've made the fix.

Reply:
Fixed it.